data/converters/converters.go

```diff
@@ -0,0 +1,195 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func toConversionError(expected string, v interface{}) error {
+	return fmt.Errorf("expected %s input but got type %T for value \"%v\"", expected, v, v)
+}
+
+// Int64NOOP The input and output values are both int64.
+var Int64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeInt64,
+}
+
+// BoolNOOP The input and output values are both bool.
+var BoolNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeBool,
+}
+
+// Float64NOOP The input and output values are both int64.
+var Float64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeFloat64,
+}
+
+// StringNOOP The input and output values are both string.
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToNullableString Converts any non-null value into a string.
```

Comment:
If it is a pointer, make it so in the comment.
Also lower case converts.
While it is nullable, the go type type is nil, so we use that instead for the comment.

// AnyToNullableString converts any non-nil value into a *string.
// If the the input value is nil the output value is *string typed nil.

and the same for following comments. (I know this seems pedantic, but these comments become the documentation as well as what people see in their IDE - and this is an SDK).

Reply:
> I know this seems pedantic

:)  for comment syntax changes, can we have lint enforce it?  There is close to zero chance I will notice casing issues in comments unless a compiler/spellcheck tells me otherwise :)